dbfit-java/informix/src/test/java/dbfit/environment/InformixBigDecimalTransformerTest.java

```diff
@@ -0,0 +1,46 @@
+package dbfit.environment;
+
+import static org.junit.Assert.*;
+import org.junit.Test;
+import org.junit.Rule;
+import org.junit.rules.ExpectedException;
+
+import java.math.BigDecimal;
+import java.lang.Double;
+import java.sql.SQLException;
+
+import dbfit.util.NormalisedBigDecimal;
+
+public class InformixBigDecimalTransformerTest {
+
+    private final InformixBigDecimalTransformer bdt = new InformixBigDecimalTransformer();
+    private final BigDecimal inBigDec = new BigDecimal("123.45");
+
+    @Test
+    public void transformsBigDecimaltoBigDecimalTest() throws SQLException {
+        BigDecimal bd = new BigDecimal(inBigDec.toString());
+        BigDecimal outBigDec = (BigDecimal) bdt.transform(bd);
+        assertEquals(outBigDec, inBigDec);
+    }
+
+    @Test
+    public void transformsNormalisedBigDecimaltoBigDecimalTest() throws SQLException {
+        NormalisedBigDecimal nbd = new NormalisedBigDecimal(inBigDec);
+        BigDecimal outBigDec = (BigDecimal) bdt.transform(nbd);
+        assertEquals(outBigDec, inBigDec);
+    }
+
+    @Rule
+    public ExpectedException exception = ExpectedException.none();
```

Comment:
Maybe it's better to move that on the top (next to the other private final attributes).